internal/cmd/config/profile/import/import.go

```diff
@@ -0,0 +1,106 @@
+package _import
```

Comment:
generally we do not have underscore _ before package names

Reply:
i used there an underscore before the package name because import is an keyword of golang and can not be used as package name. Renamed it to importProfile